openSUSE/open-build-service
# Use component to delete binaries

Use DeleteConfirmationDialogComponent instead of a partial to render the confirmation dialog.

How to test this:
- make a package build
- click on the repository name link in the build results (or either go to `/package/binaries/<project>/<package>/<repository>`)
- click on the "Delete binaries" action under the list of binaries per arch
- all the binaries of the architecture should disappear

![binaries_delete](https://user-images.githubusercontent.com/2581944/177511730-2aab29a4-5630-4186-b45a-af67fdb10b0d.png)
